docs/docs/adrs/adr-008-throttle-retries.md

```diff
@@ -21,44 +21,66 @@ For context on why the throttling mechanism exists, see [ADR 002](./adr-002-thro
+Note the consumer already queues up both `SlashPackets` and `VSCMaturedPackets` via `AppendPendingPacket`. 
+Those packets are dequeued in every `EndBlock` in `SendPackets` and sent to the provider.
 
-Instead, we will now introduce the following logic on endblock:
+Instead, we will now introduce the following logic on `EndBlock`:
 
-* Slash packets will always be sent to the provider once they're at the head of the queue. However, once sent, the consumer will not send any trailing vsc matured packets from the queue until the provider responds with an ack that the slash packet has been handled (ie. val was jailed). That is, slash packets block the sending of trailing vsc matured packets in the consumer queue.
-* If two slash packets are at the head of the queue, the consumer will send the first slash packet, and then wait for a success ack from the provider before sending the second slash packet. This seems like it'd simplify implementation.
-* VSC matured packets at the head of the queue (ie. NOT trailing a slash packet) can be sent immediately, and do not block any other packets in the queue, since the provider always handles them immediately.
+* Slash packets will always be sent to the provider once they're at the head of the queue. 
+  However, once sent, the consumer will not send any trailing `VSCMaturedPackets` from the queue until the provider responds with an ack that the `SlashPacket` has been handled (ie. validator was jailed). 
+  That is, `SlashPackets` block the sending of trailing `VSCMaturedPackets` in the consumer queue.
+* If two `SlashPackets` are at the head of the queue, the consumer will send the first `SlashPacket`, and then wait for a success ack from the provider before sending the second `SlashPacket`. 
+  This seems like it'd simplify implementation.
+* `VSCMaturedPackets` at the head of the queue (ie. NOT trailing a `SlashPacket`) can be sent immediately, and do not block any other packets in the queue, since the provider always handles them immediately.
 
-To prevent the provider from having to keep track of what slash packets have been rejected, the consumer will have to retry the sending of slash packets over some period of time. This can be achieved with an on-chain consumer param. The suggested param value would probably be 1/2 of the provider's `SlashMeterReplenishmentPeriod`, although it doesn't matter too much as long as the param value is sane.
+To prevent the provider from having to keep track of what `SlashPackets` have been rejected, the consumer will have to retry the sending of `SlashPackets` over some period of time. 
+This can be achieved with an on-chain consumer param, i.e., `RetryDelayPeriod`. 
+The suggested param value would probably be 1/2 of the provider's `SlashMeterReplenishmentPeriod`, although it doesn't matter too much as long as the param value is sane.
```

Comment:
The consumer does not know about the `ShashMeterReplenishmentPeriod`.

The word "sane" is open to interpretation, maybe consider different phrasing.

Reply:
I would argue that the consumer knows about this param as it can be queried on the provider. The `ShashMeterReplenishmentPeriod` is not something that will be change regularly. I agree with replacing the word "sane" though.

Reply:
Actually, the recommendation to set `RetryDelayPeriod` to 1/2 of `SlashMeterReplenishmentPeriod` it's weird, especially as in the code both defaults are 1 hour. So I changed to this:
> To reduce the amount of redundant re-sends, we recommend setting `RetryDelayPeriod ~ SlashMeterReplenishmentPeriod`, i.e., waiting for the provider slash meter to be replenished before resending the rejected `SlashPacket`.